configs/docker_config.py

```diff
@@ -47,7 +47,13 @@
     'SAML_ASSERTION_ENCRYPTED',
     'OFFLINE_MODE',
     'REMOTE_USER_LOGOUT_URL',
-    'REMOTE_USER_COOKIES'
+    'REMOTE_USER_COOKIES',
+    'SESSION_TYPE',
+    'SESSION_REDIS',
+    'SESSION_MEMCACHED',
+    'SESSION_MONGODB',
+    'SESSION_SQLALCHEMY',
+    'SESSION_SQLALCHEMY_TABLE'
```

Comment:
Each session type here requires an extra driver for making the connection. Can you add a [docs](https://github.com/ngoduykhanh/PowerDNS-Admin/tree/master/docs) for it? An addition `requirements-extras.txt` also good I think.

Reply:
that's a good point about drivers, I hadn't considered that.  I was mainly thinking that if filesystem was added, it made sense to add the others.  However, I'm probably not the best person to test the other options, I will remove them.  If they are required they can be added later.